pom.xml

```diff
@@ -53,6 +53,7 @@
     <pipeline-model-definition.version>1.6.0</pipeline-model-definition.version>
     <slf4jVersion>1.7.26</slf4jVersion>
     <useBeta>true</useBeta>
+    <configuration-as-code.version>1.36</configuration-as-code.version>
```

Comment:
They're up to 1.39 now.

Reply:
I'll just pick up whatever is in BOM, I don't think we need anything particular that is in 1.39